index.html

```diff
@@ -0,0 +1,218 @@
+                src="./assets/images/exhibition-icon.png"
+                alt="exhibition"
+                class="icon"
+              />
+              <p class="program-name">CC Exhibition</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry 
+             by covering not only basic and applied research,
+             but also philosophical and ethical issues
+              regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/forum-icon.png" alt="forum" class="icon" />
+              <p class="program-name">Forum</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical 
+             issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/workshop-icon.png"
+                alt="workshop"
+                class="icon"
+              />
+              <p class="program-name">Workshop</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical
+              issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/ignite-icon.png" alt="ignite" class="icon" />
+              <p class="program-name">CC Ignite</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+              both academy and industry by covering
+               not only basic and applied research,
+              but also philosophical and ethical
+               issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+        </ul>
+        <div class="whole-program"><a href="#">SEE THE WHOLE PROGRAM</a></div>
+        <button type="button" class="join">Join Artificial Intelligence
+          International Conference</button>
+      </section>
+
+      <section class="featured-speakers">
+        <h2 class="heading">Featured Speakers</h2>
+        <hr class="short-underline" />
+        <ul class="speakers"></ul>
+        <div class="button-container">
+          <button type="button" class="more">SEE MORE <i class="fa fa-chevron-down"></i></button>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
```

Comment:
### [Optional] 
This footer looks too high, which is kinda different from the design, but yours is having more content so I can't really make it a hard requirement, but it would be nice to make it a little bit low so that it will look more professional.